### ISSUE TITLE
[cppget][improvement] Importable target list in package page

### ISSUE THREAD

**Issue by Klaim** [build2/build2#186]
It would be helpful if cppget.org automatically displayed the list of exported targets of a package (at a specific version).
Most packages README currently will expose the usage of the library from the original repository because most library packages aren't using `build2` directly, so they don't clarify which targets should be used for what purpose.

I suspect this might be difficult due to the nature of the export system, so maybe an alternative would be an optional field in the manifest that would be used to list targets that users should be able to import. It could even be checked easily (though maybe with a cost). That way packages authors have a pre-defined way to clarify which targets should be used for which purpose.

**Comment by boris-kolpackov** [comment]
Yes, doing it automatically currently is not possible since most export stubs these days just try to export whatever you ask them for. One idea is to provide an "add on" documentation entry/file in the manifest (in addition to `description`/`description-file`; maybe `usage`/`usage-file` that would provide instructions on how to use the package). But listing the targets explicitly in the manifest is also an interesting idea.
